src/makeCalculator.js

```diff
@@ -5,6 +5,41 @@
  */
 function makeCalculator() {
   // write code here
+  return {
+    result: 0,
+    add(calculator, number) {
+      calculator.result += number;
+
+      return calculator;
+    },
+    subtract(calculator, number) {
+      calculator.result -= number;
+
+      return calculator;
```

Comment:
[NAMING] - The `subtract` method should not take `calculator` as a parameter. Use `this` to refer to the calculator object.